cmd/podman/tree.go

```diff
@@ -0,0 +1,176 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/containers/libpod/cmd/podman/cliconfig"
+	"github.com/containers/libpod/cmd/podman/libpodruntime"
+	"github.com/containers/libpod/libpod/image"
+	units "github.com/docker/go-units"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	treeCommand cliconfig.TreeValues
+
+	treeDescription = "Prints layer hierarchy of an image in a tree format"
+	_treeCommand    = &cobra.Command{
+		Use:   "tree",
+		Short: "Show all child images in tree structure",
+		Long:  treeDescription,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			treeCommand.InputArgs = args
+			treeCommand.GlobalFlags = MainGlobalOpts
+			return treeCmd(&treeCommand)
+		},
+		Example: "podman image tree alpine:latest",
+	}
+)
+
+func init() {
+	treeCommand.Command = _treeCommand
+	treeCommand.SetUsageTemplate(UsageTemplate())
+	treeCommand.Flags().Bool("whatrequires", false, "Show all child images in tree structure")
```

Comment:
Can we rephrase the description to "Show all child images and layers of the specified image"?